docs/hello-neighbour.yaml

```diff
@@ -361,46 +337,56 @@ paths:
         400:
           description: "Invalid input"
           schema:
-            $ref: "#/definitions/ErrorResponse"
-      
-  /user/{userUuid}:
+            $ref: "#/definitions/Error"
+            
+  /user/login:
```

Comment:
This should probably be either `/login`. While it is a user logging in, there is no interaction with a user object as such. We are essentially creating a session object.